app/Http/Controllers/BiosController.php

```diff
@@ -0,0 +1,144 @@
+<?php namespace Symposium\Http\Controllers;
+
+use Symposium\Services\CreateBioForm;
+use Auth;
+use Bio;
+use Illuminate\Support\Collection;
+use Input;
+use Log;
+use Redirect;
+use Session;
+use Validator;
+use View;
+
+use Symposium\Exceptions\ValidationException;
+
+class BiosController extends BaseController
+{
+    // @todo: Why is this here but only used for update? Why are they not using the same pattern?
+    protected $validation = [
+    ];
+
+    public function index()
+    {
+        $bios = Bio::where('user_id', Auth::user()->id)
+            ->orderBy('nickname')
+            ->get();
```

Comment:
Should use a `bios` relation on the `User` instead of looking up bios based on `user_id`. Leaky abstraction, etc.